helm-charts/common/tei/templates/deployment.yaml

```diff
@@ -54,14 +54,22 @@ spec:
             seccompProfile:
               type: RuntimeDefault
           image: huggingface/downloader:0.17.3
-          command: ['sh', '-c']
+          command: ['sh', '-ec']
           args:
             - |
-              huggingface-cli download --cache-dir /data --token $(HF_TOKEN) $(MODEL_ID);
+              echo "Huggingface log in ...";
+              huggingface-cli login --token $(HF_TOKEN);
+              echo "Download model $(MODEL_ID) ... ";
+              huggingface-cli download --cache-dir /data $(MODEL_ID);
```

Comment:
Minor suggestion, will it be better for people to understand the logic if the download and change file mode command can always use the same variable to represent model name? Now is sometimes MODEL_ID and sometimes .Values.xxxx which makes me feel that the two variables might be different somewhere.

Reply:
done